make/autoconf/lib-vmath.m4

```diff
@@ -0,0 +1,105 @@
+#
+# Copyright (c) 2023, Arm Limited. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+###############################################################################
+#
+# Setup vmath framework and check its dependences
+#
+AC_DEFUN_ONCE([LIB_SETUP_VMATH],
+[
+  AC_ARG_WITH(libsleef, [AS_HELP_STRING([--with-libsleef],
+      [specify prefix directory for the libsleef library
+      (expecting the libraries under PATH/lib and the headers under PATH/include)])])
+
+  LIBSLEEF_FOUND=no
+  LIBVMATH_CFLAGS=
+  LIBVMATH_LIBS=
+
+  if test "x${with_libsleef}" = xno; then
+    AC_MSG_NOTICE([libvmath will not be compiled, because its dependence libsleef is disabled in command line])
+  else
+    # Check the specified libsleef.so
+    if test "x${with_libsleef}" != x; then
+      AC_MSG_CHECKING([the specified LIBSLEEF])
+      if test -e ${with_libsleef}/lib/libsleef.so &&
+         test -e ${with_libsleef}/include/sleef.h; then
+        LIBSLEEF_FOUND=yes
+        LIBVMATH_LIBS="-L${with_libsleef}/lib"
```

Comment:
This should be LIBSLEEF_LIBS and ...CFLAGS.

Reply:
Seems as above. The target library is `libvmath.so`, and the flags/libs are used for building it instead of `libsleef.so`.